cpp/bench/ann/src/raft/raft_ann_bench_utils.h

```diff
@@ -100,7 +101,7 @@ class shared_raft_resources {
   ~shared_raft_resources() noexcept { rmm::mr::set_current_device_resource(orig_resource_); }
```

Comment:
There are a few build failures in CI caused by `set_current_device_resource` still requiring `rmm::mr::device_memory_resource*`. Do we need to pin a spicific RMM branch to make this work?

Reply:
Fixed.